PyInstaller/utils/win32/icon.py

```diff
@@ -169,8 +180,21 @@ def splitter(s):
     else:
         logger.info("Updating icons from %s to %s", srcpath, dstpath)
 
+    try:
+        # Attempt to load the .ico or .exe containing the icon into memory
+        # using the same mechanism as if it were a DLL. If this fails for
+        # any reason (for example if the file does not exist or is not a
+        # .ico/.exe) then LoadLibraryEx returns a null handle and win32api
+        # raises a unique exception with a win error code and a string.
+        hsrc = win32api.LoadLibraryEx(srcpath, 0, LOAD_LIBRARY_AS_DATAFILE)
+    except win32api.error as W32E:
+        # We could continue with no icon (i.e. just return) however it seems
+        # best to terminate the build with a message.
+        raise SystemExit(
+            "Unable to load icon file {}\n    {} (Error code {})".format(
+                srcpath, W32E.strerror, W32E.winerror )
+                        )
```

Comment:
Did you run flake8 on this? I would bet on this being rejected (though not sure).

Reply:
I did not. Is the spacing a problem? I didn't think PEP8 had anything to say about parameter lists. Or [this topic](https://www.python.org/dev/peps/pep-0008/#id17) looks applicable.

Reply:
Flake8 violations:

```
Found violations: PyInstaller/utils/win32/icon.py
	E203 @ 92:38 - whitespace before ':'
	E202 @ 195:54 - whitespace before ')'
	E126 @ 196:25 - continuation line over-indented for hanging indent
```

Reply:
Thanks, will fix these tomorrow

Reply:
Have installed flake8 and run it. It finds like 50 errors in that module, mostly in original code. Even if I run with `--select=E126,E202,E203` it finds two other E202's in legacy code. I will fix the ones you quote and leave the rest.

Reply:
Yes, the original code has many errors; the entire purpose of this is to slowly eliminate them over time.

Reply:
OK re-reading [the fine manual](https://github.com/pyinstaller/pyinstaller/blob/develop/doc/development/pull-requests.rst) and a [tutorial on amending and rebasing](https://www.atlassian.com/git/tutorials/rewriting-history) I am just confused and quite afraid of losing the work I've already done, or otherwise messing up the history of the pull request. So I am just going to <sigh> close this pull request and do-over with a new one, ensuring that the changed lines pass flake8.

Reply:
Git is awesome if you know how to use it. However, it seems that it is *really* confusing to *a lot* of people.

Reply:
Yeah well it is possible to be comfy in a very circumscribed workflow routine (sheesh, I've built 10+ repos comprising many K lines of original code and hundreds of commits), and then be completely confuzzled outside that round. Let's continue this discussion (as time permits) at #3097.